### DISCUSSION TITLE
How to do DAP analysis with multiple controls and multiple treatments

### DISCUSSION THREAD

**Discussion by danielcgingerich** [stuart-lab/signac#468] [Q&A] [answered]
Hello, 

I have 10 control samples and 10 treatment samples.  I want to do DAP analysis between control and treatment.  

I see a few different ways of approaching this:
1) `RunTFIDF `on all 20 samples separately, then pool control and pool treatment for DAP analysis
2) Merge control datasets, `RunTFIDF `on merged data // merge treatment datasets, `RunTFIDF `on merged data.  Then do DAP analysis
3) Integrate controls, Integrate treatment, then do DAP analysis.  I realize that integration assumes dependencies between control and treatment, but in this case, I am integrating control with control, and treatment with treatment, thus assuming no dependencies across conditions

**Comment by timoast** [comment, marked as answer]
I either 1 or 2 seem fine to me, you could try both ways and compare them. If you include batch as a latent variable in the differential accessibility model, I wouldn't expect there to be much of a difference in the results. I would not do option 3.
